browser/src/StructuralVariantList/structuralVariantTableColumns.tsx

```diff
@@ -178,30 +179,11 @@ const structuralVariantTableColumns = [
 
 export default structuralVariantTableColumns
 
-const getContextType = (context: any) => {
-  if (context.transcript_id) {
-    return 'transcript'
-  }
-  if (context.gene_id) {
-    return 'gene'
-  }
-  return 'region'
-}
-
-export const getColumnsForContext = (context: any) => {
-  const contextType = getContextType(context)
+export const getColumnsForContext = (context: Context) => {
   const columns = structuralVariantTableColumns
     .filter(
-      (column) =>
-        // @ts-expect-error TS(2554) FIXME: Expected 1 arguments, but got 2.
-        column.shouldShowInContext === undefined || column.shouldShowInContext(context, contextType)
+      (column) => column.shouldShowInContext === undefined || column.shouldShowInContext(context)
     )
-    .map((column) => ({
-      ...column,
-      description: (column as any).descriptionInContext
-        ? (column as any).descriptionInContext(context, contextType)
-        : (column as any).description,
-    }))
```

Comment:
Where did this logic go, was it unnecessary? 

Apologies if I missed it, I didn't seem to see anything else in this PR that moved this logic to elsewhere.

Reply:
> Where did this logic go, was it unnecessary?

It was. This file here has similar functionality to several other modules in the codebase, and was originally done by copypasta from one of those. If you look over the various column definitions in `structuralVariantTableColumns`, you'll see none of them have a `description` or `descriptionInContext` field set--meaning these lines at the end were dead weight. Another good example of why coding by copypasta is problematic.